rpcd/playbooks/elasticsearch.yml

```diff
@@ -54,6 +56,27 @@
       tags:
         - elasticsearch-pre-install
   roles:
-    - "elasticsearch"
+    - role: "ansible-elasticsearch"
+      es_instance_name: "openstack"
```

Comment:
The settings in group_vars are specific to and only for artifacting, since it will not read them from the play.  The artifacting puts us in a catch-22 as elasticsearch won't work with the variables as they're defined in group_vars.  I think we should just skip artifacting of the elasticsearch container alltogether as you mentioned in another comment.